<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-O2VAEQR" failures="2" tests="3" name="Standalone Test" time="82.791" errors="0" timestamp="2024-09-04T12:53:07 IST">
  <testcase classname="pageobjects.StandaloneTest" name="submitordertest" time="20.874">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at pageobjects.StandaloneTest.submitordertest(StandaloneTest.java:34)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:831)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- submitordertest -->
  <testcase classname="pageobjects.StandaloneTest" name="submitordertest" time="12.09">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at pageobjects.StandaloneTest.submitordertest(StandaloneTest.java:34)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:831)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- submitordertest -->
  <testcase classname="pageobjects.StandaloneTest" name="OrderHistoryTest" time="0.0">
    <skipped/>
  </testcase> <!-- OrderHistoryTest -->
</testsuite> <!-- Standalone Test -->
